### PR TITLE
Update React - Adds SVG <use> & <symbol> to the JSX IntrinsicElements

### PR DESCRIPTION
**Pull request by long-lazuli**
<use> & <symbol> were missing from the definition.
- documentation can be found here : https://developer.mozilla.org/en-US/docs/Web/SVG/Element/symbol & https://developer.mozilla.org/en-US/docs/Web/SVG/Element/use

### CODE REVIEW

**Comment by pspeter3** [comment]
lgtm

**Comment by vsiao** [comment]
Please add these to the `ReactDOM` interface as well.

**Comment by long-lazuli** [comment]
@vsiao The ReactDOM interface is updated as well.

**Comment by vsiao** [comment]
Oh, I'm sorry. They're not actually available on `React.DOM` (http://facebook.github.io/react/docs/tags-and-attributes.html#svg-elements), but JSX is ok because it uses `React.createElement`.

File: react/react-0.13.3.d.ts [hunk 1]
```diff
@@ -2032,7 +2036,9 @@ declare namespace JSX {
         radialGradient: React.SVGAttributes;
         rect: React.SVGAttributes;
         stop: React.SVGAttributes;
+        symbol: React.SVGAttributes;
         text: React.SVGAttributes;
         tspan: React.SVGAttributes;
+        use: React.SVGAttributes;
```
Comment (vsiao) [inline react/react-0.13.3.d.ts]: please revert changes to `react-0.13.3.d.ts`

**Pull request merged**